### PR TITLE
ipf: Fix some typos

### PR DESCRIPTION
**Pull request by ElyesH**

### CODE REVIEW

File: sbin/ipf/ipnat/ipnat.5 [hunk 1]
```diff
@@ -401,7 +401,7 @@ feature can only be combined with
 .B round-robin
 and the use of comma.
 .PP
-For TCP and UDP packets, it is possible to both match on the destiantion
+For TCP and UDP packets, it is possible to both match on the destianation
```
Comment (grahamperrin) [inline sbin/ipf/ipnat/ipnat.5:404]:
```suggestion
For TCP and UDP packets, it is possible to both match on the destination
```

File: sbin/ipf/ipf/ipf.5 [hunk 2]
```diff
@@ -890,7 +890,7 @@ construction of filter rules easier, it is possible to place them in groups.
 A rule can be both a member of a group and the head of a new group.
 .PP
 Using filter groups requires at least two rules: one to be in the group
-one one to send matchign packets to the group. If a packet matches a
+one one to send matching packets to the group. If a packet matches a
 filtre rule that is a group head but does not match any of the rules
```
Comment (grahamperrin) [inline sbin/ipf/ipf/ipf.5:894]:
```suggestion
filter rule that is a group head but does not match any of the rules
```

File: sbin/ipf/ipf/ipf.5 [hunk 3]
```diff
@@ -1430,7 +1430,7 @@ sets the size of the in-kernel log buffer in bytes.
 log_suppress
 when set, IPFilter will check to see if the packet it is logging is
 similar to the one it previously logged and if so, increases
-the occurance count for that packet. The previously logged packet
+the occurence count for that packet. The previously logged packet
```
Comment (grahamperrin) [inline sbin/ipf/ipf/ipf.5:1433]:
```suggestion
the occurrence count for that packet. The previously logged packet
```

File: sbin/ipf/ipf/ipf.5 [hunk 4]
```diff
@@ -1467,7 +1467,7 @@ when the fill percentage of the NAT table exceeds this mark, more
 aggressive flushing is enabled.
 .HP
 nat_table_wm_low
-this sets the percentage at which the NAT table's agressive flushing
+this sets the percentage at which the NAT table's aggressive flushing
 will turn itself off at.
```
Comment (grahamperrin) [inline sbin/ipf/ipf/ipf.5:1471]:
```suggestion
will turn itself off.
```

Not misspelt, but grammatically wrong (given the 'at' in the preceding line).

File: sbin/ipf/ipnat/ipnat.5 [hunk 5]
```diff
@@ -401,7 +401,7 @@ feature can only be combined with
 .B round-robin
 and the use of comma.
```
Comment (grahamperrin) [inline sbin/ipf/ipnat/ipnat.5:402]:
```suggestion
and the use of a comma.
```

**Review by grahamperrin — CHANGES_REQUESTED** [review]

**Comment by ElyesH** [comment]
Done
Thx

File: sbin/ipf/ipnat/ipnat.5 (hunk 1 above)
Comment (grahamperrin) [reply sbin/ipf/ipnat/ipnat.5]: Resolved.

File: sbin/ipf/ipf/ipf.5 (hunk 2 above)
Comment (grahamperrin) [reply sbin/ipf/ipf/ipf.5]: Resolved.

File: sbin/ipf/ipf/ipf.5 (hunk 3 above)
Comment (grahamperrin) [reply sbin/ipf/ipf/ipf.5]: Resolved.

File: sbin/ipf/ipf/ipf.5 (hunk 4 above)
Comment (grahamperrin) [reply sbin/ipf/ipf/ipf.5]: Resolved.

File: sbin/ipf/ipnat/ipnat.5 (hunk 5 above)
Comment (grahamperrin) [reply sbin/ipf/ipnat/ipnat.5]: Resolved.

**Comment by grahamperrin** [comment]
Thanks. Gentle hint: if you use GitHub with a desktop browser, the option to accept/commit a suggested change (I can't recall the wording) will automatically resolve and hide.

**Comment by ElyesH** [comment]
ping

**Review by gmshake — APPROVED** [review]
Looks good to me. Thanks very much!

**Comment by bsdimp** [comment]
Thanks! Landed.

**Pull request closed without merging**